src/dsql/DdlNodes.h

```diff
@@ -466,6 +469,7 @@ class CreateAlterFunctionNode : public DdlNode
 	MetaName name;
 	bool create;
 	bool alter;
+	bool alterIndividualParameters;
```

Comment:
This variable is not needed, the node can determine full or partial update looking whether body is set or not.

Reply:
okay

Reply:
> This variable is not needed, the node can determine full or partial update looking whether body is set or not.

okay

Reply:
Just check not only for `body` but also for `external`.

Reply:
I removed the variable and added a flag calculation based on the `body` and `external` parameters